### DISCUSSION TITLE
Loading datasets larger than memory

### DISCUSSION THREAD

**Discussion by Logon27** [jax-ml/jax#17534] [Q&A]
Hello, I was just curious if jax had any best practices or tutorials for loading datasets that are larger than memory. How is this typically accomplished? Do people normally just keep the dataset on disk and load the data in memory in batches? And is there any specific jax functionality that accomplishes this? Thanks for your time.

**Comment by chrisflesher** [comment]
This is what data loaders are for, there are some alternatives described here:
https://github.com/google/jax/discussions/16785

I've also used zarr and dask previously to do stuff like this.
